### ISSUE TITLE
draggable problem

### ISSUE THREAD

**Issue by MatneX** [nakupanda/bootstrap3-dialog#116]
Draggable Dialogs can move outside the screen.
Sometimes it results in a endless scrolling screen, sometimes the modal disapears. You should add a function for not moving draggable modals outside the active view/window. That qould be perfect, i think.

**Comment by nakupanda** [comment]
Yes, it's still imperfect, I had noticed similar problem before.

Will try to fix your mentioned one first anyhow.
